buildspec/linuxTests.yml

```diff
@@ -29,7 +31,27 @@ phases:
                     npm run testCompile
                     npm run lint
                 fi
-            - xvfb-run npm test --silent
+            - |
+                {
+                    set -o pipefail
+                    _PROMISE_REJ_MSG='rejected promise not handled'
+                    mkfifo testout
+                    (cat testout &)
+                    xvfb-run npm test --silent | tee testout \
+                        | grep -v 'undefined..reading..range' \
+                        | grep -v CODEWHISPERER_INLINE_UPDATE_LOCK_KEY \
```

Comment:
Ignoring these known issues (https://github.com/aws/aws-toolkit-vscode/issues/4196) so we can drive the others down to zero and enable the CI failure.